_data/people.yaml

```diff
@@ -53,3 +53,14 @@
     - Glacier damage identification
     - SAR data
     - Glacial process modelling 
+    
+ - name: Joey Smith
```

Comment:
Looks good @eejasm :) The build is failing because you have an extra space in front of `-` here

Reply:
Has that worked??

Reply:
yup, I'll finish this off :) cheers!